### PR TITLE
fix templating error

### PR DESCRIPTION
**Pull request by theVerySharpFlat**

### CODE REVIEW

File: server/templates/task-activity-new.html [hunk 1]
```diff
@@ -33,10 +33,10 @@
       type="text"
       name="Location"
       {{ if .plans }}
-      {{ $plan := index .plans $.selectedPlan }}
-      {{ if ne "" $plan.Location }}value="{{ $plan.Location }}"{{ end }}
+        {{ $plan := index .plans $.selectedPlan }}
+        {{ if ne "" $plan.Location }}value="{{ $plan.Location }}"{{ end }}
       {{ else }}
-      {{ $.latestActivity.Location }}
+        value="{{ $.latestActivity.Location }}"
```
Comment (nyiyui) [inline server/templates/task-activity-new.html]: Good catch. Please remove the indent though - the style here is to indent for HTML/CSS only, not for template engine things.

**Review by nyiyui — CHANGES_REQUESTED** [review]
cf. comment

File: server/templates/task-activity-new.html (hunk 1 above)
Comment (theVerySharpFlat) [reply server/templates/task-activity-new.html]: https://github.com/nyiyui/jks/pull/2/commits/d66be7b68dc6fc1bd3a34c0dcbb58c3734983a54

**Review by nyiyui — APPROVED** [review]
Thank you very much!

**Pull request merged**